src/mbgl/map/transform_state.hpp

```diff
@@ -84,7 +84,7 @@ class TransformState {
     NorthOrientation orientation = NorthOrientation::Upwards;
 
     // logical dimensions
-    uint16_t width = 0, height = 0;
+    uint16_t width = 1, height = 1;
```

Comment:
Any unsigned number != 0. Either this or we could go the other way and use `std::numeric_limits::max`.